src/compiler/llvm-to-backend/CMakeLists.txt

```diff
@@ -108,7 +108,9 @@ if(WITH_SSCP_COMPILER)
       TOOL spirv/LLVMToSpirvTool.cpp)
 
     # Install LLVM-SPIRV translator for llvm-to-spirv
-    set(LLVMSPIRV_BRANCH llvm_release_${LLVM_VERSION_MAJOR}0)
+    if(NOT LLVMSPIRV_BRANCH)
+      set(LLVMSPIRV_BRANCH llvm_release_${LLVM_VERSION_MAJOR}0)
```

Comment:
For me this fails when level zero backend is enabled: Upstream llvm-spirv translator does not have a release branch for the current LLVM development version, so that branch does not exist yet for 17. In that case, we should probably fall back to the `main` branch, although I'm unsure how to best do this in cmake here.

Reply:
Jup, I added that so that you can manually override it with `main`:  https://github.com/fodinabor/hipSYCL-LLVM-upstream-ci/blob/main/.github/workflows/weekly.yml#L75